Install.md Fix command

samcv opened the pull request:
Without the slash, the shell may try and select the file in the folder (depends on the shell and the shell's settings)

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/qtox/qtox/4473)
<!-- Reviewable:end -->

Diadlo commented:
Your PR is failing because of the Commit Guidlines https://github.com/qTox/qTox/blob/master/CONTRIBUTING.md#-git-commit-guidelines

Diadlo commented:
Also, if `ffmeg*` is file `cd` will fail too

zetok commented:
>c761671: docs(Install) Fix command that could glob incorrectly 

There's still missing colon after `)`

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/qtox/qtox/4473#-:-Kmw7eUXouTom153gQ0T:b-wj9w81)*
<!-- Sent from Reviewable.io -->

samcv committed:
docs(Install): Fix command that could glob incorrectly

Without the slash, the shell may try and select the ffmpeg archive
instead of the ffmpeg archive's extracted folder.

samcv commented:
Colon has been added.

zetok commented:
<img class="emoji" title=":lgtm_strong:" alt=":lgtm_strong:" align="absmiddle" src="https://reviewable.io/lgtm_strong.png" height="20" width="61"/>

---

Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/qtox/qtox/4473#-:-Kn3RdEEYpNNNkLl3VQH:b-jbsf8z)*
<!-- Sent from Reviewable.io -->
